sqlx/lexer.ts

```diff
@@ -275,7 +293,20 @@ function getValueMappings() {
 
   valueMappings[INNER_SQL_BLOCK_LEXER_TOKEN_NAMES.STATEMENT_SEPERATOR] = () => "";
   valueMappings[INNER_SQL_BLOCK_LEXER_TOKEN_NAMES.CLOSE_BLOCK] = () => "";
+  valueMappings[INNER_SQL_BLOCK_LEXER_TOKEN_NAMES.SINGLE_LINE_COMMENT] = (value: string) =>
```

Comment:
i'm a bit confused by the changes for `..._LINE_COMMENT` valueMappings - what is this changing? if it's fixing something, can you indicate with some changed test cases (and/or move to a different PR)?

Reply:
Hmm, I think they're residual from other fixes. My bad, will take them out